<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Bibliografía | Análisis e integración de información de datos biológicos mediante análisis funcional</title>
  <meta name="description" content="Bibliografía | Análisis e integración de información de datos biológicos mediante análisis funcional" />
  <meta name="generator" content="bookdown 0.16 and GitBook 2.6.7" />

  <meta property="og:title" content="Bibliografía | Análisis e integración de información de datos biológicos mediante análisis funcional" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Bibliografía | Análisis e integración de información de datos biológicos mediante análisis funcional" />
  
  
  

<meta name="author" content="Juan Cruz Rodriguez" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="6-cap-conclusiones.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />











<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Prefacio</a></li>
<li class="chapter" data-level="1" data-path="1-cap-af.html"><a href="1-cap-af.html"><i class="fa fa-check"></i><b>1</b> Análisis funcional</a><ul>
<li class="chapter" data-level="1.1" data-path="1-cap-af.html"><a href="1-cap-af.html#sec:ontologias"><i class="fa fa-check"></i><b>1.1</b> Ontologías</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-cap-af.html"><a href="1-cap-af.html#gene-ontology"><i class="fa fa-check"></i><b>1.1.1</b> Gene Ontology</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-cap-af.html"><a href="1-cap-af.html#otras-ontologías"><i class="fa fa-check"></i><b>1.1.2</b> Otras ontologías</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-cap-af.html"><a href="1-cap-af.html#sec:af"><i class="fa fa-check"></i><b>1.2</b> Metodologías de Análisis Funcional (AF)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-cap-af.html"><a href="1-cap-af.html#sec:asr"><i class="fa fa-check"></i><b>1.2.1</b> Análisis de Sobre-Representación (ASR)</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-cap-af.html"><a href="1-cap-af.html#puntuación-funcional-de-clase-pfc"><i class="fa fa-check"></i><b>1.2.2</b> Puntuación Funcional de Clase (PFC)</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-cap-af.html"><a href="1-cap-af.html#análisis-de-enriquecimiento-modular"><i class="fa fa-check"></i><b>1.2.3</b> Análisis de Enriquecimiento Modular</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-cap-af.html"><a href="1-cap-af.html#comentarios-finales"><i class="fa fa-check"></i><b>1.3</b> Comentarios finales</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-cap-ngs.html"><a href="2-cap-ngs.html"><i class="fa fa-check"></i><b>2</b> Fuentes de datos biológicas</a><ul>
<li class="chapter" data-level="2.1" data-path="2-cap-ngs.html"><a href="2-cap-ngs.html#sec:technols"><i class="fa fa-check"></i><b>2.1</b> Tecnologías de obtención de expresión biológica</a><ul>
<li class="chapter" data-level="2.1.1" data-path="2-cap-ngs.html"><a href="2-cap-ngs.html#microarreglos-de-adn"><i class="fa fa-check"></i><b>2.1.1</b> Microarreglos de ADN</a></li>
<li class="chapter" data-level="2.1.2" data-path="2-cap-ngs.html"><a href="2-cap-ngs.html#itraq"><i class="fa fa-check"></i><b>2.1.2</b> iTRAQ</a></li>
<li class="chapter" data-level="2.1.3" data-path="2-cap-ngs.html"><a href="2-cap-ngs.html#secuenciación-de-arn"><i class="fa fa-check"></i><b>2.1.3</b> Secuenciación de ARN</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="2-cap-ngs.html"><a href="2-cap-ngs.html#sec:repos"><i class="fa fa-check"></i><b>2.2</b> Repositorios de datos de expresión</a></li>
<li class="chapter" data-level="2.3" data-path="2-cap-ngs.html"><a href="2-cap-ngs.html#condiciones-experimentales"><i class="fa fa-check"></i><b>2.3</b> Condiciones experimentales</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-cap-ngs.html"><a href="2-cap-ngs.html#sec:bcSubt"><i class="fa fa-check"></i><b>2.3.1</b> Cáncer de mama</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-cap-ngs.html"><a href="2-cap-ngs.html#cáncer-de-próstata"><i class="fa fa-check"></i><b>2.3.2</b> Cáncer de próstata</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-cap-ngs.html"><a href="2-cap-ngs.html#comentarios-finales-1"><i class="fa fa-check"></i><b>2.4</b> Comentarios finales</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html"><i class="fa fa-check"></i><b>3</b> Análisis Funcional Integrador</a><ul>
<li class="chapter" data-level="3.1" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#motivación"><i class="fa fa-check"></i><b>3.1</b> Motivación</a></li>
<li class="chapter" data-level="3.2" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#validación-de-resultados"><i class="fa fa-check"></i><b>3.2</b> Validación de resultados</a></li>
<li class="chapter" data-level="3.3" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#datos-de-entrada"><i class="fa fa-check"></i><b>3.3</b> Datos de entrada</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#matrices-de-expresión"><i class="fa fa-check"></i><b>3.3.1</b> Matrices de expresión</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#conjuntos-de-genes"><i class="fa fa-check"></i><b>3.3.2</b> Conjuntos de genes</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#sec:algosComparados"><i class="fa fa-check"></i><b>3.4</b> Algoritmos y parámetros comparados</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#análisis-de-sobre-representación"><i class="fa fa-check"></i><b>3.4.1</b> Análisis de Sobre-Representación</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#puntuación-funcional-de-clase"><i class="fa fa-check"></i><b>3.4.2</b> Puntuación Funcional de Clase</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#resultados"><i class="fa fa-check"></i><b>3.5</b> Resultados</a><ul>
<li class="chapter" data-level="3.5.1" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#genes-diferencialmente-expresados-por-base-de-datos"><i class="fa fa-check"></i><b>3.5.1</b> Genes diferencialmente expresados por base de datos</a></li>
<li class="chapter" data-level="3.5.2" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#análisis-de-estabilidad-de-enriquecimiento"><i class="fa fa-check"></i><b>3.5.2</b> Análisis de estabilidad de enriquecimiento</a></li>
<li class="chapter" data-level="3.5.3" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#análisis-de-profundidad-en-gene-ontology"><i class="fa fa-check"></i><b>3.5.3</b> Análisis de profundidad en Gene Ontology</a></li>
<li class="chapter" data-level="3.5.4" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#sec:consensusSection"><i class="fa fa-check"></i><b>3.5.4</b> Análisis de consenso</a></li>
<li class="chapter" data-level="3.5.5" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#enriquecimiento-exclusivo-y-relevancia-de-términos"><i class="fa fa-check"></i><b>3.5.5</b> Enriquecimiento exclusivo y relevancia de términos</a></li>
<li class="chapter" data-level="3.5.6" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#sec:ifa"><i class="fa fa-check"></i><b>3.5.6</b> Análisis Funcional Integrador</a></li>
<li class="chapter" data-level="3.5.7" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#ifa-sobre-tcga"><i class="fa fa-check"></i><b>3.5.7</b> IFA sobre TCGA</a></li>
<li class="chapter" data-level="3.5.8" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#ifa-sobre-datasets-de-cáncer-de-próstata"><i class="fa fa-check"></i><b>3.5.8</b> IFA sobre datasets de cáncer de próstata</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#conclusiones"><i class="fa fa-check"></i><b>3.6</b> Conclusiones</a><ul>
<li class="chapter" data-level="3.6.1" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#comparación-de-métodos"><i class="fa fa-check"></i><b>3.6.1</b> Comparación de métodos</a></li>
<li class="chapter" data-level="3.6.2" data-path="3-cap-ifa.html"><a href="3-cap-ifa.html#aplicación-del-ifa"><i class="fa fa-check"></i><b>3.6.2</b> Aplicación del IFA</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html"><i class="fa fa-check"></i><b>4</b> Análisis masivo e integrador de conjuntos de genes</a><ul>
<li class="chapter" data-level="4.1" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#motivación-1"><i class="fa fa-check"></i><b>4.1</b> Motivación</a></li>
<li class="chapter" data-level="4.2" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#sec:voom"><i class="fa fa-check"></i><b>4.2</b> Adaptación del IFA a otras fuentes de datos</a></li>
<li class="chapter" data-level="4.3" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#herramienta-desarrollada"><i class="fa fa-check"></i><b>4.3</b> Herramienta desarrollada</a></li>
<li class="chapter" data-level="4.4" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#validación-de-la-herramienta"><i class="fa fa-check"></i><b>4.4</b> Validación de la herramienta</a></li>
<li class="chapter" data-level="4.5" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#datos-de-entrada-1"><i class="fa fa-check"></i><b>4.5</b> Datos de entrada</a><ul>
<li class="chapter" data-level="4.5.1" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#matrices-de-expresión-1"><i class="fa fa-check"></i><b>4.5.1</b> Matrices de expresión</a></li>
<li class="chapter" data-level="4.5.2" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#conjuntos-de-genes-1"><i class="fa fa-check"></i><b>4.5.2</b> Conjuntos de genes</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#estrategias-para-el-análisis-de-eficiencia"><i class="fa fa-check"></i><b>4.6</b> Estrategias para el análisis de eficiencia</a></li>
<li class="chapter" data-level="4.7" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#resultados-1"><i class="fa fa-check"></i><b>4.7</b> Resultados</a><ul>
<li class="chapter" data-level="4.7.1" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#migsa-sobre-datasets-de-microarreglos"><i class="fa fa-check"></i><b>4.7.1</b> MIGSA sobre datasets de microarreglos</a></li>
<li class="chapter" data-level="4.7.2" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#exploración-de-los-ptf-de-los-subtipos-de-cáncer-de-mama"><i class="fa fa-check"></i><b>4.7.2</b> Exploración de los PTF de los subtipos de cáncer de mama</a></li>
<li class="chapter" data-level="4.7.3" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#integración-de-resultados-de-migsa-de-tcga-y-haibe-kains"><i class="fa fa-check"></i><b>4.7.3</b> Integración de resultados de MIGSA de TCGA y Haibe-Kains</a></li>
<li class="chapter" data-level="4.7.4" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#exploración-de-los-resultados-de-tcga-en-conjunto-con-los-ptf"><i class="fa fa-check"></i><b>4.7.4</b> Exploración de los resultados de TCGA en conjunto con los PTF</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="4-cap-migsa.html"><a href="4-cap-migsa.html#conclusiones-1"><i class="fa fa-check"></i><b>4.8</b> Conclusiones</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-cap-desafios.html"><a href="5-cap-desafios.html"><i class="fa fa-check"></i><b>5</b> Desafíos que surgieron durante el trabajo de tesis</a><ul>
<li class="chapter" data-level="5.1" data-path="5-cap-desafios.html"><a href="5-cap-desafios.html#eficiencia-computacional-del-algoritmo-mgsz"><i class="fa fa-check"></i><b>5.1</b> Eficiencia computacional del algoritmo <code>mGSZ</code></a><ul>
<li class="chapter" data-level="5.1.1" data-path="5-cap-desafios.html"><a href="5-cap-desafios.html#optimización-del-algoritmo-mgsz"><i class="fa fa-check"></i><b>5.1.1</b> Optimización del algoritmo <code>mGSZ</code></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-cap-conclusiones.html"><a href="6-cap-conclusiones.html"><i class="fa fa-check"></i><b>6</b> Conclusiones y trabajo futuro</a></li>
<li class="chapter" data-level="" data-path="bibliografía.html"><a href="bibliografía.html"><i class="fa fa-check"></i>Bibliografía</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Análisis e integración de información de datos biológicos mediante análisis funcional</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="bibliografía" class="section level1 unnumbered">
<h1>Bibliografía</h1>

<div id="refs" class="references">
<div>
<p>Ashburner, M., Ball, C. A., Blake, J. A., Botstein, D., Butler, H., Cherry, J. M., … others. (2000). Gene ontology: Tool for the unification of biology. <em>Nature Genetics</em>, <em>25</em>(1), 25.</p>
</div>
<div>
<p>Atchley, D. P., Albarracin, C. T., Lopez, A., Valero, V., Amos, C. I., Gonzalez-Angulo, A. M., … Arun, B. K. (2008). Clinical and pathologic characteristics of patients with brca-positive and brca-negative breast cancer. <em>Journal of Clinical Oncology</em>, <em>26</em>(26), 4282–4288.</p>
</div>
<div>
<p>Aure, M. R., Vitelli, V., Jernström, S., Kumar, S., Krohn, M., Due, E. U., … others. (2017). Integrative clustering reveals a novel split in the luminal a subtype of breast cancer with impact on outcome. <em>Breast Cancer Research</em>, <em>19</em>(1), 44.</p>
</div>
<div>
<p>Barrett, T., Wilhite, S. E., Ledoux, P., Evangelista, C., Kim, I. F., Tomashevsky, M., … others. (2012). NCBI geo: Archive for functional genomics data sets—update. <em>Nucleic Acids Research</em>, <em>41</em>(D1), D991–D995.</p>
</div>
<div>
<p>Bastien, R. R., Rodrı'guez-Lescure, Á., Ebbert, M. T., Prat, A., Munárriz, B., Rowe, L., … others. (2012). PAM50 breast cancer subtyping by rt-qPCR and concordance with standard clinical molecular markers. <em>BMC Medical Genomics</em>, <em>5</em>(1), 1.</p>
</div>
<div>
<p>Berkeley, C. (2004). Linear models and empirical bayes methods for assessing differential expression in microarray experiments. <em>E-Book Available at Http://Www. Bepress. Com/Sagmb/Vol3/Iss1/Art3.[PubMed]</em>.</p>
</div>
<div>
<p>Bittner, M. (2005). Expression project for oncology (expo). <em>National Center for Biotechnology Information. Available at: Http://Www. Ncbi. Nlm. Nih. Gov/Geo</em>.</p>
</div>
<div>
<p>Bonnefoi, H., Potti, A., Delorenzi, M., Mauriac, L., Campone, M., Tubiana-Hulin, M., … others. (2007). RETRACTED: Validation of gene signatures that predict the response of breast cancer to neoadjuvant chemotherapy: A substudy of the eortc 10994/big 00-01 clinical trial. Elsevier.</p>
</div>
<div>
<p>Bos, P. D., Zhang, X. H.-F., Nadal, C., Shu, W., Gomis, R. R., Nguyen, D. X., … others. (2009). Genes that mediate breast cancer metastasis to the brain. <em>Nature</em>, <em>459</em>(7249), 1005.</p>
</div>
<div>
<p>Burns, P. (2011). <em>The r inferno</em>. Lulu. com.</p>
</div>
<div>
<p>Cantor, S. B., Bell, D. W., Ganesan, S., Kass, E. M., Drapkin, R., Grossman, S., … others. (2001). BACH1, a novel helicase-like protein, interacts directly with brca1 and contributes to its dna repair function. <em>Cell</em>, <em>105</em>(1), 149–160.</p>
</div>
<div>
<p>Canuel, V., Rance, B., Avillach, P., Degoulet, P., &amp; Burgun, A. (2014). Translational research platforms integrating clinical and omics data: A review of publicly available solutions. <em>Briefings in Bioinformatics</em>, <em>16</em>(2), 280–290.</p>
</div>
<div>
<p>Carlson, M., Falcon, S., Pages, H., &amp; Li, N. (2013). Org. Hs. Eg. Db: Genome wide annotation for human. R package version.</p>
</div>
<div>
<p>Cheang, M. C., Chia, S. K., Voduc, D., Gao, D., Leung, S., Snider, J., … others. (2009). Ki67 index, her2 status, and prognosis of patients with luminal b breast cancer. <em>JNCI: Journal of the National Cancer Institute</em>, <em>101</em>(10), 736–750.</p>
</div>
<div>
<p>Chen, R., &amp; Snyder, M. (2013). Promise of personalized omics to precision medicine. <em>Wiley Interdisciplinary Reviews: Systems Biology and Medicine</em>, <em>5</em>(1), 73–82.</p>
</div>
<div>
<p>Chin, K., DeVries, S., Fridlyand, J., Spellman, P. T., Roydasgupta, R., Kuo, W.-L., … others. (2006). Genomic and transcriptional aberrations linked to breast cancer pathophysiologies. <em>Cancer Cell</em>, <em>10</em>(6), 529–541.</p>
</div>
<div>
<p>Cleveland, W. S. (2001). Data science: An action plan for expanding the technical areas of the field of statistics. <em>International Statistical Review</em>, <em>69</em>(1), 21–26.</p>
</div>
<div>
<p>Consortium, G. O. (2016). Expansion of the gene ontology knowledgebase and resources. <em>Nucleic Acids Research</em>, <em>45</em>(D1), D331–D338.</p>
</div>
<div>
<p>Cooper, K., &amp; Torczon, L. (2011). <em>Engineering a compiler</em>. Elsevier.</p>
</div>
<div>
<p>Creighton, C. J., Chen, F., Zhang, Y., Gibbons, D. L., Deneen, B., Kwiatkowski, D., &amp; Ittmann, M. (2018). Pan-cancer molecular classes transcending tumor lineage across 32 cancer types, multiple data platforms, and over 10,000 cases. <em>Clinical Cancer Research</em>, clincanres–3378.</p>
</div>
<div>
<p>Curtis, C., Shah, S. P., Chin, S.-F., Turashvili, G., Rueda, O. M., Dunning, M. J., … others. (2012). The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups. <em>Nature</em>, <em>486</em>(7403), 346.</p>
</div>
<div>
<p>Dai, X., Li, T., Bai, Z., Yang, Y., Liu, X., Zhan, J., &amp; Shi, B. (2015). Breast cancer intrinsic subtype classification, clinical use and future trends. <em>American Journal of Cancer Research</em>, <em>5</em>(10), 2929.</p>
</div>
<div>
<p>Davis, A. P., Grondin, C. J., Lennon-Hopkins, K., Saraceni-Richards, C., Sciaky, D., King, B. L., … Mattingly, C. J. (2014). The comparative toxicogenomics database’s 10th year anniversary: Update 2015. <em>Nucleic Acids Research</em>, gku935.</p>
</div>
<div>
<p>Dedeurwaerder, S., Desmedt, C., Calonne, E., Singhal, S. K., Haibe-Kains, B., Defrance, M., … others. (2011). DNA methylation profiling reveals a predominant immune component in breast cancers. <em>EMBO Molecular Medicine</em>, <em>3</em>(12), 726–741.</p>
</div>
<div>
<p>Edelman, E., Porrello, A., Guinney, J., Balakumaran, B., Bild, A., Febbo, P. G., &amp; Mukherjee, S. (2006). Analysis of sample set enrichment scores: Assaying the enrichment of sets of genes for individual samples in genome-wide expression profiles. <em>Bioinformatics</em>, <em>22</em>(14), e108–e116.</p>
</div>
<div>
<p>Ein-Dor, L., Kela, I., Getz, G., Givol, D., &amp; Domany, E. (2005). Outcome signature genes in breast cancer: Is there a unique set? <em>Bioinformatics</em>, <em>21</em>(2), 171–178.</p>
</div>
<div>
<p>Ein-Dor, L., Zuk, O., &amp; Domany, E. (2006). Thousands of samples are needed to generate a robust gene list for predicting outcome in cancer. <em>Proceedings of the National Academy of Sciences</em>, <em>103</em>(15), 5923–5928.</p>
</div>
<div>
<p>Falcon, S., &amp; Gentleman, R. (2006). Using gostats to test gene lists for go term association. <em>Bioinformatics</em>, <em>23</em>(2), 257–258.</p>
</div>
<div>
<p>Fang, H., &amp; Gough, J. (2014). Thednet’approach promotes emerging research on cancer patient survival. <em>Genome Medicine</em>, <em>6</em>(8), 64.</p>
</div>
<div>
<p>Farmer, P., Bonnefoi, H., Becette, V., Tubiana-Hulin, M., Fumoleau, P., Larsimont, D., … others. (2005). Identification of molecular apocrine breast tumours by microarray analysis. <em>Breast Cancer Research</em>, <em>7</em>(2), P2–11.</p>
</div>
<div>
<p>Fernandez, E. A., &amp; Casares, F. M. (2018). Current challenges for big omics data analytics and precision medicine. <em>Med Sci Tech</em>, <em>59</em>, 1–3.</p>
</div>
<div>
<p>Fernández, E., Alvarez, M., Podhajcer, O., &amp; Stolovitzky, G. (2007). Genomica funcional: En busca de la función de los genes. <em>National Academy of Science, Argentine</em>, <em>13</em>, 63–76.</p>
</div>
<div>
<p>Fresno, C., &amp; Fernández, E. A. (2013). RDAVIDWebService: A versatile r interface to david. <em>Bioinformatics</em>, <em>29</em>(21), 2810–2811.</p>
</div>
<div>
<p>Fresno, C., González, G. A., Merino, G. A., Flesia, A. G., Podhajcer, O. L., Llera, A. S., &amp; Fernández, E. A. (2016). A novel non-parametric method for uncertainty evaluation of correlation-based molecular signatures: Its application on pam50 algorithm. <em>Bioinformatics</em>, <em>33</em>(5), 693–700.</p>
</div>
<div>
<p>Fresno, C., González, G. A., Merino, G. A., Rodriguez, J. C., Balzarini, M. G., &amp; Fernández, E. A. (2014). Control de calidad de microarreglos de adn mediante descomposición anova-pca/pls. <em>XIX Reunión Científica Del Grupo Argentino de Biometría</em>.</p>
</div>
<div>
<p>Fresno, C., Llera, A. S., Girotti, M. R., Valacco, M. P., López, J. A., Podhajcer, O. L., … Fernández, E. A. (2012). The multi-reference contrast method: Facilitating set enrichment analysis. <em>Computers in Biology and Medicine</em>, <em>42</em>(2), 188–194.</p>
</div>
<div>
<p>Gendoo, D. M., Ratanasirigulchai, N., Schröder, M. S., Paré, L., Parker, J. S., Prat, A., &amp; Haibe-Kains, B. (2015). Genefu: An r/bioconductor package for computation of gene expression-based signatures in breast cancer. <em>Bioinformatics</em>, <em>32</em>(7), 1097–1099.</p>
</div>
<div>
<p>Gillespie, C., &amp; Lovelace, R. (2016). <em>Efficient r programming: A practical guide to smarter programming</em>. &quot; O’Reilly Media, Inc.&quot;.</p>
</div>
<div>
<p>Goeman, J. J., &amp; Bühlmann, P. (2007). Analyzing gene expression data in terms of gene sets: Methodological issues. <em>Bioinformatics</em>, <em>23</em>(8), 980–987.</p>
</div>
<div>
<p>Goldhirsch, A., Winer, E. P., Coates, A., Gelber, R., Piccart-Gebhart, M., Thürlimann, B., … others. (2013). Personalizing the treatment of women with early breast cancer: Highlights of the st gallen international expert consensus on the primary therapy of early breast cancer 2013. <em>Annals of Oncology</em>, <em>24</em>(9), 2206–2223.</p>
</div>
<div>
<p>Grasso, C. S., Wu, Y.-M., Robinson, D. R., Cao, X., Dhanasekaran, S. M., Khan, A. P., … others. (2012). The mutational landscape of lethal castration-resistant prostate cancer. <em>Nature</em>, <em>487</em>(7406), 239.</p>
</div>
<div>
<p>Gromova, I., Gromov, P., Honma, N., Kumar, S., Rimm, D., Talman, M.-L. M., … Moreira, J. (2015). High level phgdh expression in breast is predominantly associated with keratin 5-positive cell lineage independently of malignancy. <em>Molecular Oncology</em>, <em>9</em>(8), 1636–1654.</p>
</div>
<div>
<p>Guen, V. J., Chavarria, T. E., Kröger, C., Ye, X., Weinberg, R. A., &amp; Lees, J. A. (2017). EMT programs promote basal mammary stem cell and tumor-initiating cell stemness by inducing primary ciliogenesis and hedgehog signaling. <em>Proceedings of the National Academy of Sciences</em>, 201711534.</p>
</div>
<div>
<p>Haibe-Kains, B., Desmedt, C., Loi, S., Culhane, A. C., Bontempi, G., Quackenbush, J., &amp; Sotiriou, C. (2012). A three-gene model to robustly identify breast cancer molecular subtypes. <em>Journal of the National Cancer Institute</em>, <em>104</em>(4), 311–325.</p>
</div>
<div>
<p>Hanahan, D., &amp; Weinberg, R. A. (2000). The hallmarks of cancer. <em>Cell</em>, <em>100</em>(1), 57–70.</p>
</div>
<div>
<p>Hess, K. R., Anderson, K., Symmans, W. F., Valero, V., Ibrahim, N., Mejia, J. A., … others. (2006). Pharmacogenomic predictor of sensitivity to preoperative chemotherapy with paclitaxel and fluorouracil, doxorubicin, and cyclophosphamide in breast cancer. <em>Journal of Clinical Oncology</em>, <em>24</em>(26), 4236–4244.</p>
</div>
<div>
<p>Hoadley, K. A., Yau, C., Wolf, D. M., Cherniack, A. D., Tamborero, D., Ng, S., … others. (2014). Multiplatform analysis of 12 cancer types reveals molecular classification within and across tissues of origin. <em>Cell</em>, <em>158</em>(4), 929–944.</p>
</div>
<div>
<p>Huang, D. W., Sherman, B. T., &amp; Lempicki, R. A. (2008a). Bioinformatics enrichment tools: Paths toward the comprehensive functional analysis of large gene lists. <em>Nucleic Acids Research</em>, <em>37</em>(1), 1–13.</p>
</div>
<div>
<p>Huang, D. W., Sherman, B. T., &amp; Lempicki, R. A. (2008b). Systematic and integrative analysis of large gene lists using david bioinformatics resources. <em>Nature Protocols</em>, <em>4</em>(1), 44.</p>
</div>
<div>
<p>Joshi-Tope, G., Gillespie, M., Vastrik, I., D’Eustachio, P., Schmidt, E., Bono, B. de, … others. (2005). Reactome: A knowledgebase of biological pathways. <em>Nucleic Acids Research</em>, <em>33</em>(suppl_1), D428–D432.</p>
</div>
<div>
<p>Kanehisa, M., &amp; Goto, S. (2000). KEGG: Kyoto encyclopedia of genes and genomes. <em>Nucleic Acids Research</em>, <em>28</em>(1), 27–30.</p>
</div>
<div>
<p>Kedaigle, A., &amp; Fraenkel, E. (2018). Turning omics data into therapeutic insights. <em>Current Opinion in Pharmacology</em>, <em>42</em>, 95–101.</p>
</div>
<div>
<p>Khatri, P., Sirota, M., &amp; Butte, A. J. (2012). Ten years of pathway analysis: Current approaches and outstanding challenges. <em>PLoS Computational Biology</em>, <em>8</em>(2), e1002375.</p>
</div>
<div>
<p>Kolesnikov, N., Hastings, E., Keays, M., Melnichuk, O., Tang, Y. A., Williams, E., … others. (2014). ArrayExpress update—simplifying data submissions. <em>Nucleic Acids Research</em>, <em>43</em>(D1), D1113–D1116.</p>
</div>
<div>
<p>Korkola, J. E., Blaveri, E., DeVries, S., Moore, D. H., Hwang, E. S., Chen, Y.-Y., … Waldman, F. M. (2007). Identification of a robust gene signature that predicts breast cancer outcome in independent data sets. <em>BMC Cancer</em>, <em>7</em>(1), 61.</p>
</div>
<div>
<p>Kuleshov, M. V., Jones, M. R., Rouillard, A. D., Fernandez, N. F., Duan, Q., Wang, Z., … others. (2016). Enrichr: A comprehensive gene set enrichment analysis web server 2016 update. <em>Nucleic Acids Research</em>, <em>44</em>(W1), W90–W97.</p>
</div>
<div>
<p>Kwa, M., Makris, A., &amp; Esteva, F. J. (2017). Clinical utility of gene-expression signatures in early stage breast cancer. <em>Nature Reviews Clinical Oncology</em>, <em>14</em>(10), 595.</p>
</div>
<div>
<p>Law, C. W., Chen, Y., Shi, W., &amp; Smyth, G. K. (2014). Voom: Precision weights unlock linear model analysis tools for rna-seq read counts. <em>Genome Biology</em>, <em>15</em>(2), R29.</p>
</div>
<div>
<p>Li, Q., Eklund, A. C., Juul, N., Haibe-Kains, B., Workman, C. T., Richardson, A. L., … Swanton, C. (2010). Minimising immunohistochemical false negative er classification using a complementary 23 gene expression signature of er status. <em>PloS One</em>, <em>5</em>(12), e15031.</p>
</div>
<div>
<p>Liberzon, A., Subramanian, A., Pinchback, R., Thorvaldsdóttir, H., Tamayo, P., &amp; Mesirov, J. P. (2011). Molecular signatures database (msigdb) 3.0. <em>Bioinformatics</em>, <em>27</em>(12), 1739–1740.</p>
</div>
<div>
<p>Liedtke, C., Mazouni, C., Hess, K. R., André, F., Tordai, A., Mejia, J. A., … others. (2008). Response to neoadjuvant therapy and long-term survival in patients with triple-negative breast cancer. <em>Journal of Clinical Oncology</em>, <em>26</em>(8), 1275–1281.</p>
</div>
<div>
<p>Liu, B., Shen, X., &amp; Pan, W. (2016). Integrative and regularized principal component analysis of multiple sources of data. <em>Statistics in Medicine</em>, <em>35</em>(13), 2235–2250.</p>
</div>
<div>
<p>Llera, A. S., Podhajcer, O. L., Breitenbach, M., Santini, L., Muller, B., Daneri Navarro, A., … others. (2015). Translational cancer research comes of age in latin america. In <em>American sci transl med</em> (Vol. 7, p. 319). American Association for the Advancement of Science.</p>
</div>
<div>
<p>Love, M. I., Huber, W., &amp; Anders, S. (2014). Moderated estimation of fold change and dispersion for rna-seq data with deseq2. <em>Genome Biology</em>, <em>15</em>(12), 550.</p>
</div>
<div>
<p>Lu, X., Lu, X., Wang, Z. C., Iglehart, J. D., Zhang, X., &amp; Richardson, A. L. (2008). Predicting features of breast cancer with gene expression patterns. <em>Breast Cancer Research and Treatment</em>, <em>108</em>(2), 191.</p>
</div>
<div>
<p>Manoli, T., Gretz, N., Gröne, H.-J., Kenzelmann, M., Eils, R., &amp; Brors, B. (2006). Group testing for pathway analysis improves comparability of different microarray datasets. <em>Bioinformatics</em>, <em>22</em>(20), 2500–2506.</p>
</div>
<div>
<p>McCarthy, D. J., &amp; Smyth, G. K. (2009). Testing significance relative to a fold-change threshold is a treat. <em>Bioinformatics</em>, <em>25</em>(6), 765–771.</p>
</div>
<div>
<p>Meng, C., Kuster, B., Culhane, A. C., &amp; Gholami, A. M. (2014). A multivariate approach to the integration of multi-omics datasets. <em>BMC Bioinformatics</em>, <em>15</em>(1), 162.</p>
</div>
<div>
<p>Metzger Filho, O., Ignatiadis, M., &amp; Sotiriou, C. (2011). Genomic grade index: An important tool for assessing breast cancer tumor grade and prognosis. <em>Critical Reviews in Oncology/Hematology</em>, <em>77</em>(1), 20–29.</p>
</div>
<div>
<p>Miller, L. D., Smeds, J., George, J., Vega, V. B., Vergara, L., Ploner, A., … others. (2005). An expression signature for p53 status in human breast cancer predicts mutation status, transcriptional effects, and patient survival. <em>Proceedings of the National Academy of Sciences</em>, <em>102</em>(38), 13550–13555.</p>
</div>
<div>
<p>Minn, A. J., Gupta, G. P., Padua, D., Bos, P., Nguyen, D. X., Nuyten, D., … others. (2007). Lung metastasis genes couple breast tumor size and metastatic spread. <em>Proceedings of the National Academy of Sciences</em>, <em>104</em>(16), 6740–6745.</p>
</div>
<div>
<p>Minn, A. J., Gupta, G. P., Siegel, P. M., Bos, P. D., Shu, W., Giri, D. D., … Massagué, J. (2005). Genes that mediate breast cancer metastasis to lung. <em>Nature</em>, <em>436</em>(7050), 518.</p>
</div>
<div>
<p>Mishra, P., Törönen, P., Leino, Y., &amp; Holm, L. (2014). Gene set analysis: Limitations in popular existing methods and proposed improvements. <em>Bioinformatics</em>, <em>30</em>(19), 2747–2756.</p>
</div>
<div>
<p>Natrajan, R., Weigelt, B., Mackay, A., Geyer, F. C., Grigoriadis, A., Tan, D. S., … others. (2010). An integrative genomic and transcriptomic analysis reveals molecular pathways and networks regulated by copy number aberrations in basal-like, her2 and luminal cancers. <em>Breast Cancer Research and Treatment</em>, <em>121</em>(3), 575–589.</p>
</div>
<div>
<p>Network, C. G. A., &amp; others. (2012). Comprehensive molecular portraits of human breast tumours. <em>Nature</em>, <em>490</em>(7418), 61.</p>
</div>
<div>
<p>Nishimura, D. (2001). BioCarta. <em>Biotech Software &amp; Internet Report: The Computer Software Journal for Scient</em>, <em>2</em>(3), 117–120.</p>
</div>
<div>
<p>Parker, J. S., Mullins, M., Cheang, M. C., Leung, S., Voduc, D., Vickery, T., … others. (2009). Supervised risk predictor of breast cancer based on intrinsic subtypes. <em>Journal of Clinical Oncology</em>, <em>27</em>(8), 1160.</p>
</div>
<div>
<p>Pavlidis, P., Qin, J., Arango, V., Mann, J. J., &amp; Sibille, E. (2004). Using the gene ontology for microarray data mining: A comparison of methods and application to age effects in human prefrontal cortex. <em>Neurochemical Research</em>, <em>29</em>(6), 1213–1222.</p>
</div>
<div>
<p>Pawitan, Y., Bjöhle, J., Amler, L., Borg, A.-L., Egyhazi, S., Hall, P., … others. (2005). Gene expression profiling spares early breast cancer patients from adjuvant therapy: Derived and validated in two population-based cohorts. <em>Breast Cancer Research</em>, <em>7</em>(6), R953.</p>
</div>
<div>
<p>Perou, C. M., Sørlie, T., Eisen, M. B., Van De Rijn, M., Jeffrey, S. S., Rees, C. A., … others. (2000). Molecular portraits of human breast tumours. <em>Nature</em>, <em>406</em>(6797), 747.</p>
</div>
<div>
<p>Prat, A., Parker, J. S., Karginova, O., Fan, C., Livasy, C., Herschkowitz, J. I., … Perou, C. M. (2010). Phenotypic and molecular characterization of the claudin-low intrinsic subtype of breast cancer. <em>Breast Cancer Research</em>, <em>12</em>(5), R68.</p>
</div>
<div>
<p>Prat, A., &amp; Perou, C. M. (2011). Deconstructing the molecular portraits of breast cancer. <em>Molecular Oncology</em>, <em>5</em>(1), 5–23.</p>
</div>
<div>
<p>Reis-Filho, J. S., &amp; Pusztai, L. (2011). Gene expression profiling in breast cancer: Classification, prognostication, and prediction. <em>The Lancet</em>, <em>378</em>(9805), 1812–1823.</p>
</div>
<div>
<p>Richardson, A. L., Wang, Z. C., De Nicolo, A., Lu, X., Brown, M., Miron, A., … Ganesan, S. (2006). X chromosomal abnormalities in basal-like human breast cancer. <em>Cancer Cell</em>, <em>9</em>(2), 121–132.</p>
</div>
<div>
<p>Rivals, I., Personnaz, L., Taing, L., &amp; Potier, M.-C. (2006). Enrichment or depletion of a go category within a class of genes: Which test? <em>Bioinformatics</em>, <em>23</em>(4), 401–407.</p>
</div>
<div>
<p>Robinson, M. D., McCarthy, D. J., &amp; Smyth, G. K. (2010). EdgeR: A bioconductor package for differential expression analysis of digital gene expression data. <em>Bioinformatics</em>, <em>26</em>(1), 139–140.</p>
</div>
<div>
<p>Rodriguez, J. C., González, G. A., Fresno, C., &amp; Fernández, E. A. (2015). Integrative functional analysis improves information retrieval in breast cancer. <em>Iberoamerican Congress on Pattern Recognition</em>.</p>
</div>
<div>
<p>Rodriguez, J. C., González, G. A., Fresno, C., Llera, A. S., &amp; Fernández, E. A. (2016a). Improving information retrieval in functional analysis. <em>Computers in Biology and Medicine</em>, <em>79</em>, 10–20.</p>
</div>
<div>
<p>Rodriguez, J. C., Merino, G. A., Llera, A. S., &amp; Fernández, E. A. (2019). Massive integrative gene set analysis enables functional characterization of breast cancer subtypes. <em>Journal of Biomedical Informatics</em>, 103157.</p>
</div>
<div>
<p>Rodriguez, J. C., Merino, G. A., Prato, L., Llera, A. S., &amp; Fernández, E. A. (2016b). The impact of rna-seq differential expression algorithms on over-representation analysis of gene sets. <em>ISCB Latin America 2016</em>.</p>
</div>
<div>
<p>Rodriguez, J. C., Prato, L., Llera, A. S., &amp; Fernández, E. A. (2017). Effects of rna-seq genes analysis on the over-representation analysis of gene sets. <em>XXI Congreso Argentino de Bioingeniería - SABI 2017</em>.</p>
</div>
<div>
<p>Rodriguez, J. C., Vargas, C., &amp; Fernández, E. A. (2018). ShinyWYSIWYG: A shiny what you see is what you get editor. <em>Latinamerican Conference About the Use of R in R&amp;D 2018</em>.</p>
</div>
<div>
<p>Ross-Adams, H., Lamb, A., Dunning, M., Halim, S., Lindberg, J., Massie, C., … others. (2015). Integration of copy number and transcriptomics provides risk stratification in prostate cancer: A discovery and validation cohort study. <em>EBioMedicine</em>, <em>2</em>(9), 1133–1144.</p>
</div>
<div>
<p>Saal, L. H., Johansson, P., Holm, K., Gruvberger-Saal, S. K., She, Q.-B., Maurer, M., … others. (2007). Poor prognosis in carcinoma is associated with a gene expression signature of aberrant pten tumor suppressor pathway activity. <em>Proceedings of the National Academy of Sciences</em>, <em>104</em>(18), 7564–7569.</p>
</div>
<div>
<p>Santuario-Facio, S. K., Cardona-Huerta, S., Perez-Paramo, Y. X., Trevino, V., Hernandez-Cabrera, F., Rojas-Martinez, A., … others. (2017). A new gene expression signature for triple-negative breast cancer using frozen fresh tissue before neoadjuvant chemotherapy. <em>Molecular Medicine</em>, <em>23</em>, 101.</p>
</div>
<div>
<p>Scaltriti, M., Elkabets, M., &amp; Baselga, J. (2016). Molecular pathways: AXL, a membrane receptor mediator of resistance to therapy. <em>Clinical Cancer Research</em>, clincanres–1458.</p>
</div>
<div>
<p>Schmidt, M., Böhm, D., Von Törne, C., Steiner, E., Puhl, A., Pilch, H., … Gehrmann, M. (2008). The humoral immune system has a key prognostic impact in node-negative breast cancer. <em>Cancer Research</em>, <em>68</em>(13), 5405–5413.</p>
</div>
<div>
<p>Sharma, D., Blum, J., Yang, X., Beaulieu, N., Macleod, A. R., &amp; Davidson, N. E. (2005). Release of methyl cpg binding proteins and histone deacetylase 1 from the estrogen receptor <span class="math inline">\(\alpha\)</span> (er) promoter upon reactivation in er-negative human breast cancer cells. <em>Molecular Endocrinology</em>, <em>19</em>(7), 1740–1751.</p>
</div>
<div>
<p>Shen, R., Olshen, A. B., &amp; Ladanyi, M. (2009). Integrative clustering of multiple genomic data types using a joint latent variable model with application to breast and lung cancer subtype analysis. <em>Bioinformatics</em>, <em>25</em>(22), 2906–2912.</p>
</div>
<div>
<p>Shi, L., Reid, L. H., Jones, W. D., Shippy, R., Warrington, J. A., Baker, S. C., … others. (2006). The microarray quality control (maqc) project shows inter-and intraplatform reproducibility of gene expression measurements. <em>Nature Biotechnology</em>, <em>24</em>(9), 1151.</p>
</div>
<div>
<p>Shi, Q., Zhang, C., Peng, M., Yu, X., Zeng, T., Liu, J., &amp; Chen, L. (2017). Pattern fusion analysis by adaptive alignment of multiple heterogeneous omics data. <em>Bioinformatics</em>, <em>33</em>(17), 2706–2714.</p>
</div>
<div>
<p>Silver, D. P., Richardson, A. L., Eklund, A. C., Wang, Z. C., Szallasi, Z., Li, Q., … others. (2010). Efficacy of neoadjuvant cisplatin in triple-negative breast cancer. <em>Journal of Clinical Oncology</em>, <em>28</em>(7), 1145.</p>
</div>
<div>
<p>Song, Z., Feng, C., Lu, Y., Lin, Y., &amp; Dong, C. (2018). PHGDH is an independent prognosis marker and contributes cell proliferation, migration and invasion in human pancreatic cancer. <em>Gene</em>, <em>642</em>, 43–50.</p>
</div>
<div>
<p>Sotiriou, C., Wirapati, P., Loi, S., Harris, A., Fox, S., Smeds, J., … others. (2006). Gene expression profiling in breast cancer: Understanding the molecular basis of histologic grade to improve prognosis. <em>Journal of the National Cancer Institute</em>, <em>98</em>(4), 262–272.</p>
</div>
<div>
<p>Subramanian, A., Tamayo, P., Mootha, V. K., Mukherjee, S., Ebert, B. L., Gillette, M. A., … others. (2005). Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. <em>Proceedings of the National Academy of Sciences</em>, <em>102</em>(43), 15545–15550.</p>
</div>
<div>
<p>Sørlie, T., Borgan, E., Myhre, S., Vollan, H. K., Russnes, H., Zhao, X., … Rødland, E. (2010). The importance of gene-centring microarray data. <em>The Lancet Oncology</em>, <em>11</em>(8), 719–720.</p>
</div>
<div>
<p>Taylor, B. S., Schultz, N., Hieronymus, H., Gopalan, A., Xiao, Y., Carver, B. S., … others. (2010). Integrative genomic profiling of human prostate cancer. <em>Cancer Cell</em>, <em>18</em>(1), 11–22.</p>
</div>
<div>
<p>Tian, L., Greenberg, S. A., Kong, S. W., Altschuler, J., Kohane, I. S., &amp; Park, P. J. (2005). Discovering statistically significant pathways in expression profiling studies. <em>Proceedings of the National Academy of Sciences</em>, <em>102</em>(38), 13544–13549.</p>
</div>
<div>
<p>Valentin, M. D., Da Silva, S. D., Privat, M., Alaoui-Jamali, M., &amp; Bignon, Y.-J. (2012). Molecular insights on basal-like breast cancer. <em>Breast Cancer Research and Treatment</em>, <em>134</em>(1), 21–30.</p>
</div>
<div>
<p>Van’t Veer, L. J., Dai, H., Van De Vijver, M. J., He, Y. D., Hart, A. A., Mao, M., … others. (2002). Gene expression profiling predicts clinical outcome of breast cancer. <em>Nature</em>, <em>415</em>(6871), 530.</p>
</div>
<div>
<p>Varambally, S., Yu, J., Laxman, B., Rhodes, D. R., Mehra, R., Tomlins, S. A., … others. (2005). Integrative genomic and proteomic analysis of prostate cancer reveals signatures of metastatic progression. <em>Cancer Cell</em>, <em>8</em>(5), 393–406.</p>
</div>
<div>
<p>Vaske, C. J., Benz, S. C., Sanborn, J. Z., Earl, D., Szeto, C., Zhu, J., … Stuart, J. M. (2010). Inference of patient-specific pathway activities from multi-dimensional cancer genomics data using paradigm. <em>Bioinformatics</em>, <em>26</em>(12), i237–i245.</p>
</div>
<div>
<p>Verhaak, R. G., Hoadley, K. A., Purdom, E., Wang, V., Qi, Y., Wilkerson, M. D., … others. (2010). Integrated genomic analysis identifies clinically relevant subtypes of glioblastoma characterized by abnormalities in pdgfra, idh1, egfr, and nf1. <em>Cancer Cell</em>, <em>17</em>(1), 98–110.</p>
</div>
<div>
<p>Waddell, N., Arnold, J., Cocciardi, S., Da Silva, L., Marsh, A., Riley, J., … others. (2010). Subtypes of familial breast tumours revealed by expression and copy number profiling. <em>Breast Cancer Research and Treatment</em>, <em>123</em>(3), 661–677.</p>
</div>
<div>
<p>Wang, B., Mezlini, A. M., Demir, F., Fiume, M., Tu, Z., Brudno, M., … Goldenberg, A. (2014). Similarity network fusion for aggregating data types on a genomic scale. <em>Nature Methods</em>, <em>11</em>(3), 333.</p>
</div>
<div>
<p>Wang, Y., Klijn, J. G., Zhang, Y., Sieuwerts, A. M., Look, M. P., Yang, F., … others. (2005). Gene-expression profiles to predict distant metastasis of lymph-node-negative primary breast cancer. <em>The Lancet</em>, <em>365</em>(9460), 671–679.</p>
</div>
<div>
<p>Weinstein, J. N., Collisson, E. A., Mills, G. B., Shaw, K. R. M., Ozenberger, B. A., Ellrott, K., … others. (2013). The cancer genome atlas pan-cancer analysis project. <em>Nature Genetics</em>, <em>45</em>(10), 1113.</p>
</div>
<div>
<p>Wickham, H. (2014). <em>Advanced r</em>. Chapman; Hall/CRC.</p>
</div>
<div>
<p>Wolfe, M. J. (1996). <em>High performance compilers for parallel computing</em>. Addison-Wesley.</p>
</div>
<div>
<p>Yersal, O., &amp; Barutca, S. (2014). Biological subtypes of breast cancer: Prognostic and therapeutic implications. <em>World Journal of Clinical Oncology</em>, <em>5</em>(3), 412.</p>
</div>
<div>
<p>Yu, K., Ganesan, K., Tan, L. K., Laban, M., Wu, J., Zhao, X. D., … others. (2008). A precisely regulated gene expression cassette potently modulates metastasis and survival in multiple solid cancers. <em>PLoS Genetics</em>, <em>4</em>(7), e1000129.</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="6-cap-conclusiones.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["RodriguezJC_tesis_doctoral.pdf", "PDF"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
